libraries/ms-common/inc/stm32f0xx/gpio_cfg.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
Fair, I went with _cfg largely because I found including a _mcu file on x86 to be kind of weird and _cfg felt more generic. _mcu to me implies that the file is specific to an MCU whereas _cfg conveys information about the configuration and setup on a particular machine. Thoughts?